src/jaegertracing/net/IPAddress.cpp

```diff
@@ -36,6 +37,12 @@ struct IfAddrDeleter : public std::function<void(ifaddrs*)> {
 
 IPAddress IPAddress::localIP(int family)
 {
+    try {
+        return versionFromString(platform::hostname(), 0, family);
+    } catch (const std::exception& e) {
```

Comment:
If you don't care about the exception, why not use `catch (,..)` to catch everything?